src/base/bittorrent/tracker.cpp

```diff
@@ -32,11 +32,16 @@
 #include <vector>
 #include <libtorrent/bencode.hpp>
 #include <libtorrent/entry.hpp>
+#include <limits>
 
 #include "base/preferences.h"
 #include "base/http/server.h"
 #include "base/utils/string.h"
 #include "tracker.h"
+#include <boost/asio/ip/address.hpp>
+#if defined(BOOST_ASIO_DYN_LINK) || !defined(BOOST_ASIO_SEPARATE_COMPILATION)
+#include <boost/asio/impl/src.hpp>
```

Comment:
Get that out of here. If qBittorrent is not compiled you have, it is the problem of your build configuration.